pyproject.toml

```diff
@@ -29,6 +29,7 @@ dependencies = [
     "pyyaml",
     "tables",
     "kafkit",
+    "nest_asyncio"
```

Comment:
This dependency seems to be gone now (🎉👏) but this dependency should now therefore be removed 🙂

Reply:
Sure!